Omie.Api.Client/Request/Customer/GetManyParameters.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+using Omie.Api.Client.Abstractions;
+
+namespace Omie.Api.Client.Request.CustomerParameters {
+    /// <remarks/>
+    public sealed class GetManyParameters : IRequestParameter {
+        /// <remarks/>
+        [JsonProperty("pagina")]
+        public int Page { get; set; }
+
+        /// <remarks/>
+        [JsonProperty("registros_por_pagina")]
+        public int Limit { get; set; }
```

Comment:
`LimitPerPage` maybe?

Reply:
From the top of mind *PageLimit* seems more usual.
I think we can use the Page prefix since the class name don't give that clue.